integration/util_test.go

```diff
@@ -722,8 +723,14 @@ func rebootContainerd(t *testing.T, sh *shell.Shell, customContainerdConfig, cus
 	)
 
 	// cleanup directories
-	testutil.KillMatchingProcess(sh, "containerd")
+	err := testutil.KillMatchingProcess(sh, "containerd")
+	if err != nil {
+		sh.Fatal("failed to kill containerd: %v", err)
+	}
 	testutil.KillMatchingProcess(sh, "soci-snapshotter-grpc")
```

Comment:
Think we need to reassign `err` here. May also consider refactoring this error check and the preceding error check to check the errors inline since there's no other output from the `KillMatchingProcess` function

```suggestion
	err = testutil.KillMatchingProcess(sh, "soci-snapshotter-grpc")
```

Reply:
Whoops, will fix.